### PR TITLE
Batch - error on missing modules

### PR DESCRIPTION
**Pull request by robinschmid**
In CLI we keep the default behavior of cancelling all.

In the BatchMode dialog if queue is imported - just show error messages for all missing modules.

### CODE REVIEW

**Comment by robinschmid** [comment]
Not sure feels like dangerous to let people run a batch file where parts are missing modules

File: src/main/java/io/github/mzmine/modules/batchmode/BatchQueue.java [hunk 1]
```diff
@@ -142,9 +141,13 @@ public static BatchQueue loadFromXml(final Element xmlElement,
           moduleFound = MZmineCore.getModuleInstance(
               (Class<MZmineModule>) Class.forName(methodName));
         } catch (ClassNotFoundException e) {
-          logger.warning(
-              "Module not found for class " + methodName + " (maybe recreate the batch file)");
-          throw new UnknownModuleNameException(methodName, e);
+          String warning =
+              "Module not found for class " + methodName + " (maybe recreate the batch file)";
```
Comment (SteffenHeu) [inline src/main/java/io/github/mzmine/modules/batchmode/BatchQueue.java:145]: maybe add the version of the batch and the mzmine version to this error message to give users a better hint what may be wrong.

Comment (robinschmid) [reply src/main/java/io/github/mzmine/modules/batchmode/BatchQueue.java]: Done

File: src/main/java/io/github/mzmine/modules/batchmode/BatchQueue.java [hunk 2]
```diff
@@ -141,8 +142,14 @@ public static BatchQueue loadFromXml(final Element xmlElement,
...
               (Class<MZmineModule>) Class.forName(methodName));
         } catch (ClassNotFoundException e) {
-          String warning =
-              "Module not found for class " + methodName + " (maybe recreate the batch file)";
+          String batchVersionStr =
+              batchMzmineVersion == null ? "of unspecified version" : batchMzmineVersion.toString();
+
+          String warning = """
+              Module not found for class %s (maybe recreate the batch file).
+              Current MZmine version: %s (batch was created with MZmine %s.""".formatted(methodName,
```
Comment (SteffenHeu) [inline src/main/java/io/github/mzmine/modules/batchmode/BatchQueue.java:150]: close bracket

Comment (robinschmid) [reply src/main/java/io/github/mzmine/modules/batchmode/BatchQueue.java]: done

**Pull request merged**